packages/nc-gui/components/dashboard/TreeView.vue

```diff
@@ -389,6 +389,34 @@ const setIcon = async (icon: string, table: TableType) => {
     message.error(await extractSdkResponseErrorMsg(e))
   }
 }
+
+const duplicateTable = async (table: TableType) => {
+  if (!table || !table.id || !table.project_id) return
+
+  const isOpen = ref(true)
+
+  const { close } = useDialog(resolveComponent('DlgTableDuplicate'), {
+    'modelValue': isOpen,
+    'table': table,
+    'onOk': async (jobData: { name: string; id: string }) => {
```

Comment:
missing try catch error handling and $e - see duplicateProject for reference.

Reply:
I moved error handling part to component (wrapping api call) as it should be on both